### PR TITLE
Solo 217 add lis3dh blocks

### PR DESCRIPTION
**Pull request by MatzElectronics**
Still a work in progress, but this should be enough to allow others on the team to test them.

### CODE REVIEW

File: src/globals.js [hunk 1]
```diff
@@ -31,7 +31,7 @@
  *
  * @type {{}}
  */
```
Comment (zfi) [inline src/globals.js]: This change is unrelated to the task. Necessary overall, but unrelated.

Comment (MatzElectronics) [reply src/globals.js]: Sorry about that - I needed to change this to make things on my dev work, and I forgot to revert it.  Do you want me to revert & commit and publish this as its own PR?

Comment (MatzElectronics) [reply src/globals.js]: Nevermind - I see that you got it.

**Comment by zfi** [comment]
This PR addresses the task created by issue #217
